README.md

```diff
@@ -41,7 +41,69 @@ npm run dev
 ### 1단계 - 웹 성능 테스트
 1. 웹 성능예산은 어느정도가 적당하다고 생각하시나요
 
+``
+* First Byte Time : 웹 서버에서 받은 컨텐츠의 첫 번째 바이트가 얼마만에 도착했는가?
+* Keep-Alive Enabled : TCP 연결을 재사용하기 위한 Keep-Alive 설정이 되어 있는가?
+* Compress Transfer : 스크립트 파일이 Content-Encoding으로 압축되어 있는가?
+* Compress Image : 이미지를 압축했는가?
+* Cache Static Content : 정적 파일이 캐싱 설정이 되어 있는가?
+* Time to Interactive(TTI) : 하위 리소스가 로드되고 사용자 입력에 응답할 수 있는 시점까지 걸린 시간.
+  * [TTI](https://web.dev/tti/)
+* Total BLocking Time(TBT) : FCP 와 TTI 사이에 총 걸린 시간.
+  * [TBT](https://web.dev/tbt/)
+* Speed Index : 사용자가 볼수 있는 컨텐츠를 랜더링하는데 걸리는 속도 (낮을수록 좋음).
+  * [Speed Index](https://web.dev/speed-index/)
+* Largest Contentful Paint(LCP) : 가장 큰 이미지 or 텍스트를 렌더링하는데 걸린 시간.
+  * [LCP](https://web.dev/lcp/)
+* First Contentful Paint(FCP) : 페이지 콘텐츠의 일부가 화면에 렌더링 될때까지의 시간.
+  * [FCP](https://web.dev/fcp/)
+  
+#### 타사 비교 (First View, 데스크톱)
+단위: 초(s)
+
+### 트래픽이 가장 많을걸로 판단되는 메인과 경로검색 페이지의 성능 지표 탐색
+
+#### 메인 페이지
+![img_1.png](img_1.png)
+
+| | [nextstep subway](http://ljh0326.p-e.kr:8080/) |
+|----------------|-------------------------------|
+|FCP|2.7|
+|TTI|2.8|
+|Speed Index|2.7|
+|LCP|2.9|
+|TBT|0.07|
+
+#### 경로 검색 페이지
+![img.png](img.png)
+
+| |[네이버 지도](https://map.naver.com/v5/directions/-/-/-/transit?c=14137575.5199888,4524330.9973991,15,0,0,0,dh)|[카카오지도](https://map.kakao.com/)|[nextstep subway](http://ljh0326.p-e.kr:8080/)|
+|----------------|-------------------------------|-----------------------------|-----------------------------|
+|FCP|0.389|0.741|3.0|
+|TTI|3.2|2.5|3.0|
+|Speed Index|1.931|3.878|3.0|
+|LCP|3.975|2.141|3.0|
+|TBT|0.021|0.059|0.01|
+
+#### 웹 성능 예산 [nextstep subway](http://ljh0326.p-e.kr:8080/)
+|목표|메인|경로검색|
+|----------------|-------------------------------|-------------------------------|
+|FCP|0.7|0.9|
+|TTI|2|2|
+|Speed Index|2|2|
+|LCP|1|1|
+|TBT|유지|
+
+이렇게 정한 이유 개선 필요한 부분과 함께 설명
+
 2. 웹 성능예산을 바탕으로 현재 지하철 노선도 서비스는 어떤 부분을 개선하면 좋을까요
+- 정적 자원 압축
+    - vendor.js (2,125KB -> 407KB) , main.js (172.0KB -> 28.1KB), stations(70KB -> 6.5KB)로 각각 압축이 가능하다. 데이터 용량이 낮아진 많큼 좀 더 사용자 요청에 응답할 수 있다.
+
+- CDN (캐시)
+    - First VIEW와 REPEAT VIEW의 전체적인 성능이 많이 차이난다. 이는 web browser의 캐싱 때문인것으로 추측하고있다. 
+      이를 통해 CDN을 통해 더 가까운 네트워크에서 캐싱된 데이터를 받는다면 웹 성능이 많이 향상될 것으로 추측된다.
```

Comment:
음.. 이 부분은 잘 모르겠네요.

<img width="1440" alt="스크린샷 2022-04-30 오후 11 07 57" src="https://user-images.githubusercontent.com/29951288/166108932-5f5c2886-5653-4b9e-8f02-6b19554283f9.png">
테스트 결과를 보면, 자바스크립트, 이미지, 폰트 등을 재요청하지 않아서 그렇다고 볼 수 있는데요. 왜 재요청을 하지 않는지까지는 모르겠네요.


> Repeat View:
The Repeat View row is a test that was done immediately after the First View test without clearing out anything. The browser window is closed after the First View test and then a new browser is launched to do the Repeat View test. The Repeat View test represents what someone will see if they are coming back to the page some time after visiting it the first time (and is a best-case scenario since the test is re-visiting the page immediately).

Repeat View 기능은 브라우저를 끄고 다시 실행하는 것이니 브라우저 캐시가 걸릴 일이 없고, 
이와 별개로, 현재 서버는 말씀하셨듯 캐싱 설정이 되어 있지 않고, 요청할 때도 no-cache 요청을 하기에 매번 재로딩 됩니다.

브라우저에서 직접 퍼포먼스 탭으로 보았을 떄도 브라우저 캐싱이 안되는 것을 확인할 수 있어요

<img width="1383" alt="스크린샷 2022-04-30 오후 11 23 06" src="https://user-images.githubusercontent.com/29951288/166109517-43a8ae41-ef4e-45c1-b882-43acf6f5e65c.png">